### PR TITLE
Update node version

### PR DESCRIPTION
**Pull request by Pespiri**

### CODE REVIEW

File: src/components/page-configuration/use-regenerate-deploy-key-and-secret.js [hunk 1]
```diff
@@ -1,9 +1,11 @@
+import { nanoid } from 'nanoid';
+
 import { usePostJson } from '../../effects';
-var phraseit = require('phraseit');
 
-const useRegenerateDeployKeyAndSecret = (appName) => {
+export const useRegenerateDeployKeyAndSecret = (appName) => {
```
Comment (nilsgstrabo) [inline src/components/page-configuration/use-regenerate-deploy-key-and-secret.js:5]: This function is only used by the configure-application-github component. Perhaps we should move it there.

**Review by nilsgstrabo — CHANGES_REQUESTED** [review]
Github is struggeling at the moment. Getting status 500 when trying to add some of the comments, so I'll add them here instead.

- In configure-application-github/index.js line 161: Do we need to set value property? This is a Checkbox, so perhaps only checked is needed.
- I think we should remove the isDismissable property from page-create-application/index.js line 93. It is really annoying that the Dialog closes when I accidentially click outside.
- In src/models/application-summary/test-data.ts line 10: We should not reference test data from other tests.
- Same as above for src/models/environment-summary/test-data.ts
- ...and src/models/environment-variable/test-data.ts
- ...and src/models/job-summary/test-data.ts
- ...and src/models/scan/test-data.ts

File: src/components/app-list/index.test.tsx [hunk 2]
```diff
@@ -1,41 +1,18 @@
...
-      created: new Date('2018-11-22T14:26:49+0000'),
-      ended: new Date('2018-11-22T14:32:15+0000'),
-      environments: ['prod', 'qa'],
-      name: 'radix-pipeline-13123123-hftd',
-      pipeline: 'build-deploy',
-      started: new Date('2018-11-22T14:26:49+0000'),
-      status: ProgressStatus.Succeeded,
-    },
-  },
+  ApplicationSummaryData[0],
```
[the remark this answers is not in the document]
Comment (Pespiri) [reply src/components/app-list/index.test.tsx]: removed dependency on testdata from the models

File: src/components/configure-application-github/index.js [hunk 3]
```diff
@@ -11,20 +11,20 @@ import { copy } from '@equinor/eds-icons';
...
+import imageDeployKey from './deploy-key02.png';
+import imageWebhook from './webhook02.png';
+
 import { Alert } from '../alert';
 import { Code } from '../code';
-import useRegenerateDeployKeyAndSecret from '../page-configuration/use-regenerate-deploy-key-and-secret';
-import externalUrls from '../../externalUrls';
-import { RequestState } from '../../state/state-utils/request-states';
-import { copyToClipboard } from '../../utils/string';
+import { useRegenerateDeployKeyAndSecret } from '../page-configuration/use-regenerate-deploy-key-and-secret';
```
[the remark this answers is not in the document]
Comment (Pespiri) [reply src/components/configure-application-github/index.js]: moved into appropriate folder

File: src/components/page-configuration/use-regenerate-deploy-key-and-secret.js (hunk 1 above)
Comment (Pespiri) [reply src/components/page-configuration/use-regenerate-deploy-key-and-secret.js]: moved into appropriate folder

**Comment by Pespiri** [comment]
> Github is struggeling at the moment. Getting status 500 when trying to add some of the comments, so I'll add them here instead.
> 
> * In configure-application-github/index.js line 161: Do we need to set value property? This is a Checkbox, so perhaps only checked is needed.
> * I think we should remove the isDismissable property from page-create-application/index.js line 93. It is really annoying that the Dialog closes when I accidentially click outside.
> * In src/models/application-summary/test-data.ts line 10: We should not reference test data from other tests.
> * Same as above for src/models/environment-summary/test-data.ts
> * ...and src/models/environment-variable/test-data.ts
> * ...and src/models/job-summary/test-data.ts
> * ...and src/models/scan/test-data.ts

kept models tests as is, but added comments
fixed the rest

**Review by nilsgstrabo — APPROVED** [review]

**Pull request merged**